lexorus/sample-ci-project
# Integrate danger-swift into project and github actions

### Background
[Danger](https://github.com/danger/danger) is a great tool to integrate with your continuous integration. I would like to have it integrated into this sample project, at least to lint the PR titles.

### What has been done
1. Add `Dangerfile.swift` to the project
2. Integrated [`danger-swift`](https://github.com/danger/swift) as part of the `Github Actions`. It will run on PR's.

### How to test
1. Change the `PR`'s title to start with lowercased letter or contain `WIP`. The warning should be posted.
2. Change the `PR`'s title to be longer than 65 characters. This should trigger the error/fail.
3. Since `Dangerfile.swift` is a swift file that is not tested, the warning about missing tests will be posted.
